docs/environment/storage.md

```diff
@@ -76,3 +76,110 @@ Note that Redis and Memcache through ElastiCache run even when not used and thus
+
+```yaml
+service: app
+provider:
+    name: aws
+    runtime: provided.al2
+plugins:
+    - ./vendor/bref/bref
+functions:
+    app:
+        handler: index.php
+        layers:
+            - ${bref:layer.php-74-fpm}
+        events:
+            - httpApi: '*'
+```
+
+As any service, the DynamoDB table can be deployed using the `resources.Resources` subkey:
+
+```yaml
+service: app
+provider:
+    name: aws
+    runtime: provided.al2
+plugins:
+    - ./vendor/bref/bref
+functions:
+    app:
+        handler: index.php
+        layers:
+            - ${bref:layer.php-74-fpm}
+        events:
+            - httpApi: '*'
+resources:
+    Resources:
+        CacheTable:
+            Type: AWS::DynamoDB::Table
+            Properties:
+              AttributeDefinitions:
+                - AttributeName: id
+                  AttributeType: S
+              BillingMode: PAY_PER_REQUEST
+              TimeToLiveSpecification:
+                AttributeName: ttl
+                Enabled: true
+              KeySchema:
+                - AttributeName: id
+                  KeyType: HASH
+```
+
+Explanation of the properties:
+
+- [AttributeDefinitions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html#cfn-dynamodb-table-attributedef) contains the list of mandatory attributes and must contain a partition key (similar to primary key in databases), which is in this example simply called `id`. The [AttributeType](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-dynamodb-attributedef.html#cfn-dynamodb-attributedef-attributename-attributetype) `S` means it's a string. Note that no other properties except the keys need to be specified as DynamoDB is a NoSQL database without a fixed schema.
+- [BillingMode](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html#cfn-dynamodb-table-billingmode) needs to be set to PAY_PER_REQUEST, otherwise you will incur costs even when no requests are arriving.
+- [TimeToLiveSpecification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html#cfn-dynamodb-table-timetolivespecification) is optional and allows you to set a field that will act as a time to live attribute, it's very useful for caching purposes.
+- [KeySchema](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html#cfn-dynamodb-table-keyschema) is a definition of the type of the primary/sort keys (not a datatype which was already defined in AttributeDefinitions). Possible values for [KeyType](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-dynamodb-keyschema.html#aws-properties-dynamodb-keyschema-keytype) are `HASH` (partition/primary key) and `RANGE` (sort/secondary key).
+
+You need to add a permission to access the DynamoDB from your function using `provider.iamRoleStatements`.
```

Comment:
Instead of `provider.iamRoleStatements` it will be better to use the new format `provider.iam.role.statements` (see https://www.serverless.com/framework/docs/providers/aws/guide/iam/)